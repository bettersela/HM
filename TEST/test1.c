//Oct  test
void main(){
	int n = 0o1000;
	print n;//  | PRINT Expr { Prim1("printi", $2) }
	println;//  | PRINTLN  { Prim1("printc", nl) }
}

