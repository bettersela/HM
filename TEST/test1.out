24 19 1 5 25 13 0 0 1 0 512 12 15 -1 13 0 0 1 11 22 15 -1 0 10 23 15 -1 15 0 21 0